flow-typed/types.js

```diff
@@ -10,6 +10,7 @@ declare type BalanceString = string
 
 declare type Address = string
 declare type OwnerAddress = Address
+declare type ContactId = OwnerAddress // It's just address yet
```

Comment:
there is specific file `flow-typed/redux/favorites`. I suggest rename it to `contacts` and place new types into it.

Reply:
I have heard that we don't plan to store contacts in redux storage

Reply:
...oneday
:D